### PR TITLE
Gives RO a taser

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

title
## Why It's Good For The Game

How hard is it towait 10 seconds for the RO to give you something. For example this screenshot is what happened after a fucking marine couldnt wait 10 seconds
image

## Changelog
:cl:
balance: The RO has a taser
/:cl:

### CODE REVIEW

**Comment by Ridgyaxe** [comment]
The problem with this PR, is that the captain opened fire on two marines first, and there was no RO. The grenades were from a result of the captain shooting.

**Comment by Hughgent** [comment]
The RO already has the FLASH level skill, do they not have access to the flash?

**Comment by TiviPlus** [comment]
It spawns in their locker apparently but god knows

And this isnt a case of "oh it was one time" but marines keep breaking into req over and over and its a pain in the ass for everyone involved

**Comment by David-Stormwell** [comment]
Check if the RO has high enough police skill for it, if not increase it.

**Comment by Chaznoodles** [comment]
> It spawns in their locker apparently but god knows
> 
> And this isnt a case of "oh it was one time" but marines keep breaking into req over and over and its a pain in the ass for everyone involved

The req lockers on PoS aren't the Requisitions Officer-specific one, that has a flash. Change them to the Requisitions Officer locker instead of the Cargo Tech one, and avoid ROs acting like world police in future.

File: code/datums/skills.dm [hunk 1]
```diff
@@ -375,7 +375,7 @@ engineer, construction, leadership, medical, surgery, pilot, police, powerloader
 	construction = SKILL_CONSTRUCTION_PLASTEEL
 	leadership = SKILL_LEAD_TRAINED
 	powerloader = SKILL_POWERLOADER_MASTER
-	police = SKILL_POLICE_FLASH
+	police = SKILL_POLICE_MP//You broke into the wrong req bucko
```
Comment (Sibg) [inline code/datums/skills.dm:378]: Looks like RO is going to do some hurting.

**Comment by neohenry** [comment]
I would prefer a taser over a flash since I can quick draw a nonlethal weapon against a disrespectful player. Then again, (1) I'm a security player and (2) I'm one of the few players that like to have a taser in the server.

Not having a taser in these rare freak incidents makes me want to make it easier for me to have a taser.

**Comment by neohenry** [comment]
#4481

**Comment by psykzz** [comment]
Consider adding tasers to the CIC armory too, so it can be paied with #4515 and #4513

**Pull request merged**